### PR TITLE
Small style/spelling changes for Train example plugin

### PR DESCRIPTION
**Pull request by jerryaldrichiii**

### CODE REVIEW

File: examples/plugins/train-local-rot13/lib/train-local-rot13/connection.rb [hunk 1]
```diff
@@ -52,11 +52,6 @@ def initialize(options)
         # credentials, now is a good time.
       end
 
-      # If you are writing a local-style connection, implement this to return true.
```
Comment (clintoncwolfe) [inline examples/plugins/train-local-rot13/lib/train-local-rot13/connection.rb]: Please explain this in a comment in lib/train/plugins/base_connection.rb . You learned a lot of tribal knowledge from Jared about why this isn't needed - record it in the superclass.

Comment (jerryaldrichiii) [reply examples/plugins/train-local-rot13/lib/train-local-rot13/connection.rb]: Oooooh, great call! I'll make that happen.

Comment (jerryaldrichiii) [reply examples/plugins/train-local-rot13/lib/train-local-rot13/connection.rb]: Better yet, I think I can just remove it all together. I'll submit a PR.

**Comment by jquick** [comment]
re-doing tests

**Comment by clintoncwolfe** [comment]
Ignoring unrelated unit test failures due to #393

**Pull request merged**